libnavigation-sku/src/main/java/com/mapbox/navigation/sku/accounts/Billing.kt

```diff
@@ -0,0 +1,59 @@
+package com.mapbox.navigation.sku.accounts
+
+import android.content.Context
+import android.content.pm.ApplicationInfo
+import android.content.pm.PackageManager
+import com.mapbox.android.accounts.v1.AccountsConstants.DEFAULT_TOKEN_MANAGE_SKU
+import com.mapbox.android.accounts.v1.AccountsConstants.KEY_META_DATA_MANAGE_SKU
+
+internal class Billing private constructor() {
+
+    enum class BillingModel {
```

Comment:
What about a `sealed class` instead? That way we get the `when` completeness in `MapboxNavigationAccounts` L#31 for free. What do you think?

Reply:
You do get `when` completeness with enums as well :)

Reply:
Not enforced by the build system 😉

Reply:
It is enforced by the compiler! :)